interface/core.xhr.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "core.xhr",
+  "api": {
+    // @param {String} method - GET/POST/DELETE
+    // @param {String} url
+    // @param {Boolean} async
```

Comment:
I don't see the value in mimicking parameters that we can't implement.  The purpose of "async=false" is to make the `send()` block until a response comes back from the server, but our `send()` will never block.

Bonus: "async=false" is [officially deprecated](https://xhr.spec.whatwg.org/#sync-warning), at least in webpages.